src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -334,14 +335,24 @@ public override bool IsBindingRequired
         {
             get
             {
-                if (MetadataKind == ModelMetadataKind.Property)
-                {
-                    return BindingMetadata.IsBindingRequired;
-                }
-                else
+                if (!_isBindingRequired.HasValue)
                 {
-                    return false;
+                    if (MetadataKind == ModelMetadataKind.Type)
+                    {
+                        _isBindingRequired = false;
```

Comment:
I don't think so. `IsBindingRequired` is special because we want to know if the user set a value or not. We don't care in this case.